cloud/pkg/admissioncontroller/admission.go

```diff
@@ -224,6 +234,7 @@ func (ac *AdmissionController) registerWebhooks(opt *options.AdmissionOptions, c
 			},
 		},
 	}
+	validatingWebhookConfigurations[ValidateDeviceModelConfigName] = deviceModelCRDWebhook
```

Comment:
yes, but for validating webhook, we only have one hook `deviceModelCRDWebhook`, and can we not register it?

Reply:
For example, Is this command OK?
`admission --webhook-configuration-names="mutate-offlinemigration"`
we don't even register the `deviceModelCRDWebhook` webhook at all. Will this work well?

Reply:
It's ok to do that. We can only register `offlineMigrationWebhook`. It will work well.